website/content/en/blog/2025-09-navigating-platform-engineering

```diff
@@ -0,0 +1,59 @@
+---
+title:  "Navigating Platform Engineering with the maturity model assessment"
+slug: navigating-platform-engineering
+date:   2025-09-02 01:00:00 +0000
+author: Steve Fenton
+categories:
+- Article
+tags:
+- WG Platforms
+- Community Contributions
+---
+
+The [CNCF Platform Engineering community](https://cloudnativeplatforms.com) has been working on an assessment to accompany the [Platform Engineering Maturity Model](https://cloudnativeplatforms.com/whitepapers/platform-eng-maturity-model/). It's now available as a preview for feedback, but it's crucial to understand its purpose to know what it can do for you and what you need to put in once you get your score.
+
+Help us out by [trying out the assessment](https://cloud-native-platform-engineering.github.io/pemm-assessment/) and [giving us feedback](https://docs.google.com/forms/d/e/1FAIpQLScXru41BVVQDipxuSCQaNmo0GcBpBM8jHhDbX3pQskJXFgV8A/viewform).
```

Comment:
I feel we should provide a brief about the assessment itself. 

Something like:
- The assessment has multiple-choice questions...
- It will take about 15-20 mins for you to complete the assessment...
- You can host this assessment in your environment....
- No responses are stored...
- At the end of the assessment, you'll see a report card that will...